packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -93,6 +96,9 @@ export class BlockList extends Component {
 					ListHeaderComponent={ header }
 					ListEmptyComponent={ this.renderDefaultBlockAppender }
 					ListFooterComponent={ withFooter && this.renderBlockListFooter }
+					getItemLayout={ ( data, index ) => {
```

Comment:
Do we still need this considering the current temporary fix? If we don't let's remove it to avoid having unexpected side effects

Reply:
Yes, it's still needed to get working properly `FloatingToolbar` on Android.